packages/vue/src/components/SpCrud/SpCrud.vue

```diff
@@ -9,31 +9,47 @@
           size="md"
           style="font-weight: 700"
       >
-        Post items
+        {{ itemName }} items
       </SpTypography>
       <SpButton
-          aria-label="Create post"
           type="primary"
-          @click="visibleModal = 'create-post'"
+          @click="visibleModal = 'create-item'"
```

Comment:
I propose to create a separate object that will be responsible for holding modal states:
```
isModalShown: any = {
    create: false,
    delete: false
};
```
and have separate props and call the state `:open="isModalShown['create']"`. States can be changed with a global function:
```
openModal(modal: string, state: boolean) {
    isModalShown[modal] = state;
}
```